pykeeb/keyboard_matrix.py

```diff
@@ -109,3 +152,53 @@ def get_matrix(self): #needs more elegant solution, maybe using __add__?
 				if column < self.columns - 1:
 					x += self.column_hulls[row][column]
 		return x
+
+	def get_plate(self): #needs more elegant solution, maybe using __add__?
+			x = Cube(0) 
+			for column in range(self.columns):
+				for row in range(self.rows):
+					x += self.sm[row][column].get_switch_at_location() 
+					if column < self.columns - 1:
+						x += self.column_hulls[row][column]
+					if row < self.rows - 1:
+						x += self.row_hulls[row][column]
+						if column < self.columns - 1:
+							x += self.corner_hulls[row][column]
+			return x
+
+
+	def get_walls(self): #needs more elegant solution, maybe using __add__?
+			x = Cube(0) 
+			for column in range(self.columns):
+				for row in range(self.rows):
+					if row == 0:
+						x += self.back_wall[column]
+						if column < self.columns - 1:
+							x += self.back_wall_hulls[column]
+						if column == 0:
+							x += self.back_left_corner
+						if column == self.columns - 1:
+							x += self.back_right_corner
+					if row == self.rows - 1:
+						x += self.front_wall[column]
+						if column < self.columns - 1:
+							x += self.front_wall_hulls[column]
+						if column == 0:
+							x += self.front_left_corner
+						if column == self.columns - 1:
+							x += self.front_right_corner
+					if column == 0:
+						x += self.left_wall[row]
+						if row < self.rows - 1:
+							x += self.left_wall_hulls[row]
+					if column == self.columns - 1:
+						x += self.right_wall[row]
+						if row < self.rows - 1:
+							x += self.right_wall_hulls[row]
+					# if row < self.rows - 1:
```

Comment:
Why are these commented out?  Dactyl specific?

Reply:
the column hulls, row hulls and corner hulls kind of make the plate a unit. I moved these lines to the get_plate function...just made more sense to me. These lines are vestigial and can be deleted. TBH, I am kind of new to GitHub and did not mean to commit these two functions to the master repo, just my branch. But if you think they provide value, I will clean them up and add docstrings.